### ISSUE TITLE
rhiot cmd should test wget command availability 

### ISSUE THREAD

**Issue by gautric** [rhiot/rhiot#355]
with docker ubuntu and fedora image,
we have to install wget to use `rhiot` cmd

**Comment by hekonsek** [comment]
Yeah. We should check it and inform user that wget has to be installed.

On Fri, Dec 18, 2015, 14:25 Greg. A. <notifications@github.com> wrote:

> with docker ubuntu and fedora image,
> we have to install wget to use rhiot cmd
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rhiot/rhiot/issues/355>.
>
-- 
Henryk Konsek
http://about.me/hekonsek

**Comment by PatrickSteiner** [comment]
+1 just ran into that this morning!

next issues when installing rhiot-cmd on Mac:

- wget only worked with "--no-check-certificate"
- still getting the following

sudo wget https://raw.githubusercontent.com/rhiot/rhiot/master/tooling/bash/rhiot --no-check-certificate -O /usr/bin/rhiot && sudo chmod +x /usr/bin/rhiot
Password:
/usr/bin/rhiot: Operation not permitted

**Comment by gautric** [comment]
for mac user, I think It could be more difficult, because apple introduces : 
```
https://developer.apple.com/library/mac/documentation/Security/Conceptual/System_Integrity_Protection_Guide/FileSystemProtections/FileSystemProtections.html
```
